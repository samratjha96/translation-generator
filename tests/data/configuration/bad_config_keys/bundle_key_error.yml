bundle:
  - path: .
    extension: properties
    default_locale: en_US
