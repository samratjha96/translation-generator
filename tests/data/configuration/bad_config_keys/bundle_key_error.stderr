Validator: bundle_key_error.yml does not have any bundles
